app/com/gitpitch/services/ImageService.java

```diff
@@ -66,8 +67,8 @@ public String buildBackgroundOffline(String md) {
             String fragUrl = frag.substring(0, frag.indexOf("\""));
             String imageName = FilenameUtils.getName(fragUrl);
             String imageUrl = IMG_OFFLINE_DIR + imageName;
-            return buildBackground(null, imageUrl);
 
```

Comment:
@jroper understood. I've actually already merged your PR as is. I've a little more work to do on related issues to make this all work seamlessly for both online and offline presentations. Once that work is completed and I've updated [GitPitch.com](https://gitpitch.com) I'll let you know.